packages/ibm-products/src/patterns/CoachmarkStacked/CoachmarkStacked.mdx

```diff
@@ -0,0 +1,38 @@
+import { Story, Controls, Source, Canvas } from '@storybook/addon-docs/blocks';
+import { CodesandboxLink } from '../../global/js/utils/story-helper';
+import * as CoachmarkStackedStories from './CoachmarkStacked.stories';
+import { stackblitzPrefillConfig } from '../../../previewer/codePreviewer';
+
+# CoachmarkStacked
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+
+## Overview
+
+The CoachmarkStacked pattern acts as a container element for onboarding and
+should only be used within the scope of a Coachmark.
+
+To build this pattern, we recommend including the following components:
+
+- [Coachmark](https://carbon-for-ibm-products.netlify.app/?path=/docs/experimental-onboarding-coachmark-next--overview)
+- [cds-button](https://web-components.carbondesignsystem.com/?path=/docs/components-button)
+- Carousel utility
```

Comment:
we should definitely link to any documentation for this utility or at least clarify what it is and where it's located